packages/vega-schema/src/axis.js

```diff
@@ -59,6 +59,11 @@ const axis = object({
   translate: numberType,
   values: arrayOrSignal,
   zindex: numberType,
+  ariaHidden: booleanValue,
```

Comment:
Not something we need to do now, but it might be helpful in the future to define repeated properties (like these aria properties) in an object bundle and use object rest destructuring to insert them where needed. There would be no major compatibility issues as this is part of the compile process only.